.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "3.6"
+  - "3.8"
```

Comment:
what's the reason behind this? The MSs that depend on this lib run 3.6, so by using 3.8 here, we are exposing ourselves to issues. Please revert this change.